src/Symfony/Component/Form/Extension/Core/Type/FormType.php

```diff
@@ -202,6 +208,8 @@ public function configureOptions(OptionsResolver $resolver)
             'invalid_message' => 'This value is not valid.',
             'invalid_message_parameters' => [],
             'is_empty_callback' => null,
+            'get' => null,
+            'set' => null,
```

Comment:
Wouldn't it be "better" to call them "getter" and "setter"?

Reply:
Works for me too. 

Indeed, it also matches the variables in the `CallbackAccessor`. I will make the changes soon, thanks!

Reply:
Done.